src/cljs/webchange/ui_framework/layout/navigation/styles.scss

```diff
@@ -1,42 +1,106 @@
-.navigation-menu {
-    .menu {
-        border-radius: 8px;
-        margin: 0;
-        padding: 0 0 0 32px;
-        width: 100%;
-        min-width: 200px;
-
-        .menu-item {
+.left-side-bar {
+    max-width: 296px;
+	width:100%;
+	height: 100%;
+	background: #353535;
+	padding:40px 24px 88px;
+	box-sizing: border-box;
+	position: fixed;
+    left:0;
+    top:0;
+    
+    img {
+        max-width: 100%;
+        vertical-align: middle;
+    }
+
+    .pad-fit-l {
+        padding-left: 16px;
+    }
+
+    .mb-lg {
+        margin-bottom: 47px;
+    }
+
+    .d-inline {
+        display: inline-block;
+    }
+
+    .nav-menu {
+
+		ul, ul li {
+            cursor: pointer;
             list-style: none;
-            border-left: solid 1px #cecece;
+            list-style-type: none;
+            margin: 0;
+            padding: 0;
+        }
 
-            &:last-child {
-                margin-bottom: 0;
+        ul li {
+            font-weight: normal;
+            font-size: 14px;
+            line-height: 20px;
+            align-items: center;
+            color: #9E9E9E;
+            margin-bottom: 8px;
+            display: table;
+            text-decoration: none;
+            padding: 10px 18px;
+            min-width: 168px;
+
+            span {
+                height: 20px;
+	            display: inline-block;
             }
 
-            button {
-                border: none !important;
-                border-radius: 4px;
-                font-size: 14px !important;
-                justify-content: flex-start;
-                height: 40px;
-                width: 100%;
-                margin-bottom: 16px;
+            span:first-child {
+                width: 20px;
+	            float: left;
             }
 
-            &.active {
-                button {
-                    background: #EAEAEA;
-                }
+            span:nth-child(2) {
```

Comment:
I would be better to give block his own class name like `user-name` and use it instead of count first or second child. 
Also you can use [nested selectors](https://sass-lang.com/documentation/style-rules#nesting) here